### PR TITLE
PBI-30: create retrieve comment

### PR DESCRIPTION
**Pull request by kanziebub**

### CODE REVIEW

File: src/Controllers/MaterialCommentController.php [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+            $user = auth()->user();
+            $request->validate([
+                'material_id' => 'required|integer',
+                'content' => 'required|string|max:65535',
+            ]);
+
+            $announcement = LessonMaterial::where('id', $request->input('material_id'))
+                ->first();
+
+            if (! $announcement) {
```
Comment (AldiNFitrah) [inline src/Controllers/MaterialCommentController.php]:
```suggestion
            $lessonMaterial = LessonMaterial::where('id', $request->input('material_id'))
                ->first();

            if (! $lessonMaterial) {
```

Comment (AldiNFitrah) [inline src/Controllers/MaterialCommentController.php]: and I think there should be a validation where user that is not in the course cannot add a comment. Perhaps using:
```php
            if (! CourseUserHelper::isUserInCourse(
                $user->id,
                $lessonMaterial?->course_id,
            ))
```

**Review by AldiNFitrah — CHANGES_REQUESTED** [review]

**Review by nofamex — APPROVED** [review]
LGTM

**Review by Zsaschz — APPROVED** [review]
LGTM

**Review by darren-ngoh — APPROVED** [review]
LGTM!!!

**Pull request merged**